qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -213,15 +216,12 @@ def jobs(
                 continue
 
             job_id = job_info.get('id', "")
-            # TODO Extract job name from job_info instead of passing it in
-            # once it becomes available from the API.
             # TODO: first argument for IBMQJob should be a Backend, but as
             # `job_responses` comes from `jobs_statuses`, the backend name
             # might not be present. Revise during IBMQJob refactoring.
             job_info.update({
                 '_backend': None,
                 'api': self._provider._api,
-                'name': job_name
```

Comment:
The new api change hasn't been deployed yet, so we might want to hold off merging this one until it does.

Reply:
Yes, no need to rush indeed :+1: